bokeh/query.py

```diff
@@ -41,7 +41,7 @@ class LEQ(object): pass
 class NEQ(object): pass
 
 
-def match(obj, selector, context={}):
+def match(obj, selector, context=None):
```

Comment:
I don't understand well enough what context is to tell (its not documented). Should context be used in the recursion or not? Since context is/was never modified in-place, I don't think this change means that it should be passed to the recursive call now.